web/wp-content/plugins/ad-stack/js/ad-stack.js

```diff
@@ -45,14 +46,16 @@ function TMBI_Ad_Stack( ) {
 		}
 	}
 
+
 	/*get the current browser width*/
 	function getCurrentBreakPoint() {
```

Comment:
This function needs to be updated to work with user-defined breakpoints. As they are filterable, we can't assume there will always be "large_screen", "tablet", "mobile" and "desktop" breakpoints. From the theme we may need a "smartwatch" breakpoint, a "4k" breakpoint, etc, and the function should be able to use them.